icons/users-2.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M14 19a6 6 0 0 0-12 0" />
+  <circle cx="8" cy="9" r="4" />
+  <path d="M16 13a4 4 0 0 0 0-8" />
+  <path d="M22 19a6 6 0 0 0-6-6" />
```

Comment:
```suggestion
  <path d="M22 19a6 6 0 0 0-6-6 4 4 0 1 0 0-8" />
```

Reply:
Oh, yeah, that would've been pretty trivial. :D